api/graphql/schema.graphql

```diff
@@ -1641,7 +1656,7 @@ type Mutation {
   updatePoolCandidateAsAdmin(
     id: ID!
     poolCandidate: UpdatePoolCandidateAsAdminInput! @spread
-  ): PoolCandidate @update @guard @can(ability: "update", find: "id")
+  ): PoolCandidate @update @guard @can(ability: "updateAsAdmin", find: "id")
```

Comment:
It's a little weird to call this policy updateAsAdmin since it's actually the Pool Operator, not the Platform Admin who calls this under the new permissions structure.  But the mutation is still called updatePoolCandidateAsAdmin so I followed that for the policy.